helix-core/src/main/java/org/apache/helix/manager/zk/ZkBaseDataAccessor.java

```diff
@@ -84,6 +88,9 @@ public AccessResult() {
   private static Logger LOG = LoggerFactory.getLogger(ZkBaseDataAccessor.class);
 
   private final HelixZkClient _zkClient;
+  // onDemand zkClient for read/write non-znRecord data path;
```

Comment:
read/write "to" non-ZnRecord data

-> remove "path" because there's no such thing as ZnRecord path or non-ZnRecord path.

Reply:
fixed